Source/Core/InputCommon/ControlReference/FunctionExpression.cpp

```diff
@@ -636,6 +636,57 @@ class PulseExpression : public FunctionExpression
   mutable Clock::time_point m_release_time = Clock::now();
 };
 
+// usage: ntimes(input, n = 1)
```

Comment:
@jordan-woyak camel case 😂?

Nitpick but it should be
`// usage: ntimes(input, times = 1)`
to be aligned with the expected arguments.

Reply:
Nitpick but it should be
// usage: ntimes(input, times = 1)
to be aligned with the expected arguments.

Reply:
D'oh, thank you. I couldn't decide whether the second argument should be called `times` or `n` and apparently went for both without noticing 🤦